out/pluralrules/diff.html

```diff
@@ -976,16 +976,43 @@
 emu-val {
```

Comment:
Nit: I prefer updating the HTML in a standalone commit (which I usually put directly on main) because it makes the diffs nicer.

Reply:
Will do it!!

Reply:
Done

Reply:
Already separated in two commits emu/html